prow/config/config.go

```diff
@@ -2474,7 +2486,6 @@ func parseProwConfig(c *Config) error {
 	for name, templates := range c.Tide.MergeTemplate {
 		if templates.TitleTemplate != "" {
 			titleTemplate, err := template.New("CommitTitle").Parse(templates.TitleTemplate)
-
```

Comment:
Let's not introduce diffs like these that are not related to the code that's being changed.

Reply:
Removed 👍